packages/wells/src/client/baseWellsClient.ts

```diff
@@ -210,6 +214,14 @@ export class ConfigureAPI {
       }
     }
 
+    if (limit) {
+      if (path.includes('?env') || path.includes('?cursor')) {
+        path += `&limit=${limit}`;
+      } else {
+        path += `?limit=${limit}`;
+      }
+    }
```

Comment:
We're starting to get some different cases here with and without `?` and `&`. Maybe it would be easier to create a javascript object with all the parameters, and then convert it to a query string?

https://stackoverflow.com/a/53171438/111471

So like this,
```js
const parameters = {env: "api", cursor: "abc123}
return path + "?" + new URLSearchParams(object).toString()
```
Something like that. Haven't tested it myself. Just an idea.

Reply:
good idea :+1: